wsdl_docs.inc

```diff
@@ -631,7 +631,12 @@ function _wsdl_docs_service_import($name) {
     return 'ERROR input file is not XML';
   }
   // Update existing service with new specs per file upload.
-  wsdl_docs_update_wsclient_service($service, $DOMDocument);
-
+  try {
+    wsdl_docs_update_wsclient_service($service, $DOMDocument);
+  }
+  catch (WSClientException $e) {
```

Comment:
This is good. Without this code, I would expect the user's client (curl, postman, etc) to get a 5xx message and complain about the server not responding. This would be because the exception wasn't handled and PHP stopped.

With this try/catch block, the user should receive a much nicer message back.